Proxy does not build on FreeBSD as of preview.1 release

As with  #1390, the build fails on FreeBSD after the addition of the FUSE dependency. The build fails as follows:

```
# github.com/hanwen/go-fuse/v2/fuse
go/pkg/mod/github.com/hanwen/go-fuse/v2@v2.1.0/fuse/types.go:550:2: undefined: Attr
go/pkg/mod/github.com/hanwen/go-fuse/v2@v2.1.0/fuse/types.go:578:2: undefined: Attr
go/pkg/mod/github.com/hanwen/go-fuse/v2@v2.1.0/fuse/api.go:224:41: undefined: GetAttrIn
go/pkg/mod/github.com/hanwen/go-fuse/v2@v2.1.0/fuse/api.go:225:41: undefined: SetAttrIn
go/pkg/mod/github.com/hanwen/go-fuse/v2@v2.1.0/fuse/api.go:228:39: undefined: MknodIn
go/pkg/mod/github.com/hanwen/go-fuse/v2@v2.1.0/fuse/api.go:252:42: undefined: SetXAttrIn
go/pkg/mod/github.com/hanwen/go-fuse/v2@v2.1.0/fuse/api.go:258:40: undefined: CreateIn
go/pkg/mod/github.com/hanwen/go-fuse/v2@v2.1.0/fuse/api.go:260:38: undefined: ReadIn
go/pkg/mod/github.com/hanwen/go-fuse/v2@v2.1.0/fuse/api.go:269:39: undefined: WriteIn
go/pkg/mod/github.com/hanwen/go-fuse/v2@v2.1.0/fuse/api.go:278:41: undefined: ReadIn
go/pkg/mod/github.com/hanwen/go-fuse/v2@v2.1.0/fuse/api.go:278:41: too many errors
```

--- a/proxy/fuse/fuse.go
+++ b/proxy/fuse/fuse.go
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:build !windows && !openbsd
-// +build !windows,!openbsd
+//go:build !windows && !openbsd && !freebsd
+// +build !windows,!openbsd,!freebsd
 
 // Package fuse provides a connection source wherein the user does not need to
 // specify which instance they are connecting to before they start the

--- a/proxy/fuse/fuse_freebsd.go
+++ b/proxy/fuse/fuse_freebsd.go
@@ -1,4 +1,4 @@
-// Copyright 2022 Google Inc. All Rights Reserved.
+// Copyright 2022 Google LLC
 //
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.

--- a/proxy/fuse/fuse_freebsd.go
+++ b/proxy/fuse/fuse_freebsd.go
@@ -1,0 +1,32 @@
+// Copyright 2022 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package fuse is a package stub for freebsd, which isn't supported by our fuse
+// library.
+package fuse
+
+import (
+	"errors"
+	"io"
+
+	"github.com/GoogleCloudPlatform/cloudsql-proxy/proxy/proxy"
+)
+
+func Supported() bool {
+	return false
+}
+
+func NewConnSrc(mountdir, tmpdir string, client *proxy.Client, connset *proxy.ConnSet) (<-chan proxy.Conn, io.Closer, error) {
+	return nil, nil, errors.New("fuse not supported on freebsd")
+}

